logrotate/CMakeLists.txt

```diff
@@ -21,12 +21,25 @@ SET(ACTIVE_CPP0xx_DIR "Release")
 INCLUDE (${LogRotate_SOURCE_DIR}/Options.cmake)
 INCLUDE_DIRECTORIES(${G3_HEADER_PATH})
 
+IF (MSVC)
+    SET(CMAKE_DEBUG_POSTFIX "_d")
+ENDIF()
+
 #Find all libraries
 
 find_package(Boost COMPONENTS filesystem system REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 
-find_library(G3LOG ${G3LOG_LIBRARY_TYPE} PATHS ${G3_LIBRARY_PATH})
+find_package( ZLIB REQUIRED )
+include_directories( ${ZLIB_INCLUDE_DIRS} )
+
+IF (MSVC)
+   find_library(G3LOG_DEBUG ${G3LOG_LIBRARY_TYPE}_d PATHS ${G3_LIBRARY_PATH_DEBUG})
+   find_library(G3LOG_RELEASE ${G3LOG_LIBRARY_TYPE} PATHS ${G3_LIBRARY_PATH_RELEASE})
```

Comment:
Hi Kjell,

Things have been busy for quite some time but I can now spare some time.
The fix for line 37 is really to add SET(CMAKE_DEBUG_POSTFIX "_d")
I am OK with that but in the end its up to you to decide if you want / like
it or not.
In windows land it is pretty common to have such a suffix, so I can easily
add something along the lines of

IF (MSVC)
SET(CMAKE_DEBUG_POSTFIX "_d")

The course of action then would be to add that first and then do a final
check that all is good in the logrotate side and then merge.

On 10 January 2016 at 17:45, Kjell Hedström <notifications@github.com>
wrote:

> In logrotate/CMakeLists.txt
> <https://github.com/KjellKod/g3sinks/pull/20#discussion_r49282778>:
>
> >  #Find all libraries
> >
> >  find_package(Boost COMPONENTS filesystem system REQUIRED)
> >  include_directories(${Boost_INCLUDE_DIRS})
> >
> > -find_library(G3LOG ${G3LOG_LIBRARY_TYPE} PATHS ${G3_LIBRARY_PATH})
> > +find_package( ZLIB REQUIRED )
> > +include_directories( ${ZLIB_INCLUDE_DIRS} )
> > +
> > +IF (MSVC)
> > +   find_library(G3LOG_DEBUG ${G3LOG_LIBRARY_TYPE}_d PATHS ${G3_LIBRARY_PATH_DEBUG})
> > +   find_library(G3LOG_RELEASE ${G3LOG_LIBRARY_TYPE} PATHS ${G3_LIBRARY_PATH_RELEASE})
>
> I'm referring to line 37
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KjellKod/g3sinks/pull/20/files#r49282778>.
>